app/services/hyrax/workflow/abstract_notification.rb

```diff
@@ -9,6 +31,10 @@ def self.send_notification(entity:, comment:, user:, recipients:)
 
       attr_reader :work_id, :title, :comment, :user, :recipients
 
+      # @param [Sipity::Entity] entity - the Sipity::Entity that is a proxy for the relevant Hyrax work
+      # @param [#comment] comment - the comment associated with the action being taken, could be a Sipity::Comment, or anything that responds to a #comment method
+      # @param [Hyrax::User] user - the user who has performed the relevant action
+      # @param [Hash] recipients - a hash with keys "to" and (optionally) "cc", each of which contains an Array of Users to notify
```

Comment:
You can also add the options to YARD, and abridge the comment about `recipients`, via:

```
# @option recipients [Array<Hyrax::User>] :to a list of users to which to send the notification
# @option recipients [Array<Hyrax::User>] :cc a list of users to which to copy on the notification
```